core-java-modules/core-java-exceptions-3/src/main/java/com/baeldung/exceptions/nosuchfielderror/NoSuchFieldError.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.exceptions.nosuchfielderror;
+
+import java.lang.reflect.Field;
+
+public class NoSuchFieldError {
```

Comment:
This might get confused with the core Java class of the same name. See Javadoc https://docs.oracle.com/javase/8/docs/api/java/lang/NoSuchFieldError.html

Reply:
agreed. Will rename it.